tests/Makefile

```diff
@@ -8,7 +8,7 @@ integration: verify-vendor test
 
 .PHONY: test
 test: ## Run only the tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path --bin-dir $(PROJECT_DIR)/bin)" ./hack/test.sh
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path --bin-dir $(PROJECT_DIR)/bin --remote-bucket openshift-kubebuilder-tools)" ./hack/test.sh
```

Comment:
can I get a comment with a link to the docs for whatever command has this --remote-bucket arg so should you win the lottery I have some shot at reverse engineering how it works. :)